app/controllers/miq_policy_controller.rb

```diff
@@ -711,6 +711,11 @@ def replace_right_cell(nodetype, replace_trees = [], presenter = nil)
     render :json => presenter.for_render
   end
 
+  # Clear out search text when clear when requested
+  def clear_search_text
+    @search_text = params[:search_text] = nil
+  end
```

Comment:
can't we move this method to application_controller, so it is not repeated and can be used by all the controllers?